src/test/scala/cognite/spark/v1/AssetsRelationTest.scala

```diff
@@ -298,6 +298,52 @@ class AssetsRelationTest extends FlatSpec with Matchers with ParallelTestExecuti
     assert(assetsRead == 1)
   }
 
+  it should "support option filter assetSubtreeIds" taggedAs ReadTest in {
+    val df = spark.read
+      .format("cognite.spark.v1")
+      .option("apiKey", readApiKey)
+      .option("type", "assets")
+      .option("limitPerPartition", "1000")
+      .option("partitions", "1")
+      .option("assetSubtreeIds", "2161493773812721")
```

Comment:
are the assets that assumed to be existing created in tbe beginning of test?

Reply:
those are assets in the magic `publicdata` project